### PR TITLE
rating-component

### PR DESCRIPTION
**Pull request by bipin7yadav**
-checked rating
-ratings in points
-without checked rating

### CODE REVIEW

**Review by someshtiwarii — APPROVED** [review]
try removing extra spaces !

File: rating.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+body{
+    position: relative;
+    margin-left: 4px;
+    width: 100%;
+}
+
+.body{
+    width: 100%;
+}
```
Comment (BelideNikhil) [inline rating.css:17]: don't use body as a class name

File: rating.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    margin-top: 3rem;
+    display: flex;
+    justify-content: space-between;
+}
+
+button{
+    border-radius:1rem ;
+    padding: 1rem;
+    background-color: var(--tertiary-color);
+}
```
Comment (BelideNikhil) [inline rating.css:65]: this will effect globally
try to create classes and then use them

**Review by BelideNikhil — APPROVED** [review]

File: rating.css (hunk 2 above)
Comment (bipin7yadav) [reply rating.css:65]: sure

File: rating.css (hunk 1 above)
Comment (bipin7yadav) [reply rating.css:17]: ok

**Pull request merged**